packages/db/src/translate/file.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiFile } from '@cumulus/types/api/files';
+
+import { PostgresFile } from '../types/file';
+
+export const translateApiFiletoPostgresFile = (
+  file: ApiFile
+): Omit<PostgresFile, 'granule_cumulus_id'> => ({
+  bucket: file.bucket,
+  checksum_type: file.checksumType,
+  checksum_value: file.checksum,
+  // TODO: do we really need both of these properties?
```

Comment:
As it stands, the `filename` and `file_name` properties exist on the PG table schema, which is an artifact of the refactoring we did to the Dynamo files model a while ago and preserving backwards compatibility. At some point it would be nice to clean up this mess, but I suppose this PR is not the time